content/tutorials/portenta-h7/vs-ard-lora/content.md

```diff
@@ -0,0 +1,157 @@
+
+![vs_ard_ttn_click_register](assets/vs_ard_ttn_click_register.png)
+
+On the registration page that shows up we must fill in two fields. The first one is the **Device ID** which this must be lowercase and without spaces. Since the ID should be unique, so you can use  the name of the board followed by the first and the last byte of the EUI. The second one is the **EUI** which is exactly the device EUI that appeared in the Serial Monitor. Copy paste the EUI from the Serial Monitor into the EUI field
+
+![vs_ard_ttn_register_device](assets/vs_ard_ttn_register_device.png)
+
+After pressing the Register button, your board will show up on the **Device Overview** page. You can now see all the information needed to complete the Arduino setup.
+
+![vs_ard_ttn_device_overview](assets/vs_ard_ttn_device_overview.png)
+
+### 5. Connecting to TTN
+
+Once your board has been registered you can send information to TTN. Let's come back to the Serial Monitor and proceed. It will ask for:
+
+* activation mode (that in this case is OTAA),
+* the Application EUI
+* the App Key.
+
+Lets start by making a connection through the OTAA. Enter "1" in the Serial Monitor input box and press ENTER. Then, find the EUI and the App key from TTN **Device Overview** page. You can read more into OTA vs ABP activation mode at [this link](https://www.thethingsnetwork.org/docs/devices/registration.html)
+
+```
+Your module version is: ARD-078 1.1.9
+Your device EUI is: a8xxxxxxxxxxxx0a
+Are you connecting via OTAA (1) or ABP (2)?
+Enter your APP EUI
+Enter your APP KEY
+```
+
+Next, introduce the **APP EUI** and the **APP KEY** in the Serial Monitor in a similar manner. If everything has gone fine, you will see this message:
+
+```
+Message sent correctly!
+```
+
+## Conclusion
+
+If you have sent the message successfuly you have configured Portenta H7 and the Lora Vision Shield on the TTN. We have retrieved the device EUI, used it to register the device in the TTN console, and programmed the board using the data provided by TTN. We can now send data over the LoRa® network which can be viewed from anywhere in the world (as long as we have an Internet connection and our device is in range from a TTN gateway).
+
+### Next Steps
+
+-   Try sending uplink and downlink messages between Portenta and your TTN application with _LoraSendAndReceive_ sketch from the MKRWAN library.
+-   Experiment your Vision Shield's capabilities with OpenMV and the examples from the dedicated library for Arduino. You can continue with [this tutorial](https://www.arduino.cc/pro/tutorials/portenta-h7/por-openmv-bt) from the Arduino Pro site.
+-   Combine LoRaWAN protocol with an OpenMV example to develop your own IoT application. Take advantage of the Vision Shield's camera to detect, filter, classify images, read QR codes or more.     
+
+## Troubleshooting
```

Comment:
General thought: Is it possible to provide different use cases in the Troubleshooting sections that entails "normal" issues that users may encounter?